Userland/Utilities/kcov-example.cpp

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2021, Patrick Meyer <git@the-space.agency>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <fcntl.h>
+#include <stdio.h>
+#include <sys/ioctl.h>
+#include <sys/ioctl_numbers.h>
+#include <sys/kcov.h>
+#include <sys/mman.h>
+#include <unistd.h>
+
+// Note: This program requires serenity to be built with the CMake build option
+// ENABLE_KERNEL_COVERAGE_COLLECTION
+int main(void)
+{
+    constexpr size_t num_entries = 1024 * 100;
+
+    int fd = open("/dev/kcov", O_RDWR);
+    if (fd == -1)
+        perror("open");
```

Comment:
We should probably also return here after an error occurred.

Reply:
ack